sourcemash/api/items.py

```diff
@@ -126,13 +145,17 @@ def get(self):
 
         trending_items = trending_items[:MAX_TRENDING_ITEMS]
 
-        return {'items': [marshal(Item.query.get(id), item_fields) for id, count in trending_items]}
+        return {'items': [marshal(Item.query.get(id), item_fields)
+                for id, count in trending_items]}
 
 
 class FeedItemListAPI(Resource):
 
     def get(self, feed_id):
-        return {'items': [marshal(item, item_fields) for item in Item.query.filter_by(feed_id=feed_id).all()]}
+        return {'items': [marshal(item, item_fields)
+                for item in Item.query.filter_by(feed_id=feed_id)
+                                      .limit(MAX_ARTICLES).all()]}
```

Comment:
It probably does this by default since ids increase, but i made the change just to be sure.